src/PKSim.Infrastructure/Serialization/ORM/Mappers/ProjectMetaDataToProjectMapper.cs

```diff
@@ -70,6 +72,10 @@ public IPKSimProject MapFrom(ProjectMetaData projectMetaData)
          //Once reference to dynamic meta data was added, deserialize the project itself
          _serializationManager.Deserialize(project, projectMetaData.Content.Data);
 
+         //if the project DB Version is the same as the current project, the project did not change
+         if (projectMetaData.Version == ProjectVersions.Current)
```

Comment:
We used to always set this flag to false, thus bypassing any information from project conversion (the message was nit shown EVEN IF project conversion happened). Now the logic is different so this needs to be adapted. The mapper is the perfect place